zerver/lib/integrations.py

```diff
@@ -341,6 +341,7 @@ def __init__(self, name: str, *args: Any, **kwargs: Any) -> None:
 
 WEBHOOK_INTEGRATIONS: list[WebhookIntegration] = [
     WebhookIntegration("airbrake", ["monitoring"]),
+    WebhookIntegration("airbyte", ["monitoring"], display_name="Airbyte"),
```

Comment:
Thank you for your comments @laurynmm. I need a little bit of clarity on this phrase `I think our general preference is to have the example fixture data generated directly from the application we're integrating with and not taken from the application's documentation.` Are you saying we need to generate live data directly from Airbyte? Please kindly clarify. Thank you.

Reply:
I think I understand what is required here for a real data generated from Airbyte after going through the zulip's [documentation](https://zulip.com/api/incoming-webhooks-walkthrough) again @laurynmm. Here are my thoughts, please correct me if I miss any step.
- I need to use a tool like [Webhook Site]( https://webhook.site/) to capture webhook payload(s) from Airbyte.
- I will then configure Airbyte webhook on Airbyte UI with the URL generated from step 1.
- Run a synchronization job in Airbyte to generate the webhook payload.
- Capture the webhook payload from [webhook.site]( https://webhook.site/) or any other tool.
- Lastly, save this payload as the fixture data.
Please, kindly confirm if I am on the right track.

Reply:
Yes, that's what I was thinking. You could also use the Zulip [JSON formatter](https://zulip.com/integrations/doc/json) integration to get the payload as well. Just have the URL set up to have the bot send you a DM vs to a channel/topic.

Reply:
Thank you. I'm on it now.

Reply:
@laurynmm I have been able to generate the data directly from Airbyte but the payload contains a lot more information that may not be needed since the `data` section that I am using. So, I was wondering if I should strip out the unwanted information and use only the section that is needed or leave it as it is for the fixture data. 
The payload looks like this:
```json
{
  "text": "Your connection Google Sheets \u2192 Postgres from Google Sheets to Postgres succeeded\nThis was for null\n\nYou can access its logs here: https://cloud.airbyte.com/workspaces/84d2dd6e-82aa-406e-91f3-bf8dbf176e69/connections/aa941643-07ea-48a2-9035-024575491720\n\nJob ID: 20441143",
  "blocks": [
    {
      "type": "section",
      "text": {
        "type": "mrkdwn",
        "text": "Sync completed: <https://cloud.airbyte.com/workspaces/84d2dd6e-82aa-406e-91f3-bf8dbf176e69/connections/aa941643-07ea-48a2-9035-024575491720|Google Sheets \u2192 Postgres>"
      }
    },
    {
      "type": "section",
      "fields": [
        {
          "type": "mrkdwn",
          "text": "*Source:*"
        },
        {
          "type": "mrkdwn",
          "text": "<https://cloud.airbyte.com/workspaces/84d2dd6e-82aa-406e-91f3-bf8dbf176e69/source/363c0ea3-e989-4051-9f54-d41b794d6621|Google Sheets>"
        },
        {
          "type": "mrkdwn",
          "text": "*Destination:*"
        },
        {
          "type": "mrkdwn",
          "text": "<https://cloud.airbyte.com/workspaces/84d2dd6e-82aa-406e-91f3-bf8dbf176e69/destination/b3a05072-e3c8-435a-8e6e-4a5c601039c6|Postgres>"
        },
        {
          "type": "mrkdwn",
          "text": "*Duration:*"
        },
        {
          "type": "mrkdwn",
          "text": "1 min 23 sec"
        }
      ]
    },
    {
      "type": "section",
      "text": {
        "type": "mrkdwn",
        "text": "*Sync Summary:*\n1400 record(s) extracted / 1400 record(s) loaded\n281 kB extracted / 281 kB loaded\n"
      }
    }
  ],
  "data": {
    "workspace": {
      "id": "84d2dd6e-82aa-406e-91f3-bf8dbf176e69",
      "name": "Zulip Airbyte Integration",
      "url": "https://cloud.airbyte.com/workspaces/84d2dd6e-82aa-406e-91f3-bf8dbf176e69"
    },
    "connection": {
      "id": "aa941643-07ea-48a2-9035-024575491720",
      "name": "Google Sheets → Postgres",
      "url": "https://cloud.airbyte.com/workspaces/84d2dd6e-82aa-406e-91f3-bf8dbf176e69/connections/aa941643-07ea-48a2-9035-024575491720"
    },
    "source": {
      "id": "363c0ea3-e989-4051-9f54-d41b794d6621",
      "name": "Google Sheets",
      "url": "https://cloud.airbyte.com/workspaces/84d2dd6e-82aa-406e-91f3-bf8dbf176e69/source/363c0ea3-e989-4051-9f54-d41b794d6621"
    },
    "destination": {
      "id": "b3a05072-e3c8-435a-8e6e-4a5c601039c6",
      "name": "Postgres",
      "url": "https://cloud.airbyte.com/workspaces/84d2dd6e-82aa-406e-91f3-bf8dbf176e69/destination/b3a05072-e3c8-435a-8e6e-4a5c601039c6"
    },
    "jobId": 20441143,
    "startedAt": "2024-10-22T20:27:59Z",
    "finishedAt": "2024-10-22T20:29:22Z",
    "bytesEmitted": 288179,
    "bytesCommitted": 288179,
    "recordsEmitted": 1400,
    "recordsCommitted": 1400,
    "errorMessage": null,
    "durationFormatted": "1 min 23 sec",
    "bytesEmittedFormatted": "281 kB",
    "bytesCommittedFormatted": "281 kB",
    "success": true,
    "durationInSeconds": 83
  }
}
```

Reply:
No, we want the actual payloads as our fixture data. So it's fine if there is information we're not using in the Zulip message notification.

Reply:
Also, if you want to use the payload examples below (https://github.com/zulip/zulip/pull/31922#pullrequestreview-2385155158) for the fixture data that would be fine as well.